google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java

```diff
@@ -471,8 +492,11 @@ WriteChannelConfiguration setProjectId(String projectId) {
       loadConfigurationPb.setClustering(clustering.toPb());
     }
     loadConfigurationPb.setUseAvroLogicalTypes(useAvroLogicalTypes);
-    return new com.google.api.services.bigquery.model.JobConfiguration()
-        .setLoad(loadConfigurationPb);
+    if (labels != null) {
+      jobConfiguration.setLabels(labels);
+    }
```

Comment:
again, no need to check for null, otherwise there will be no way to unset labels.